test/mnist/package/client/fedn.yaml

```diff
@@ -0,0 +1,7 @@
+entry_points:
+  predict:
+    command: python3 predict.py
+  train:
+    command: python3 train.py
+  validate:
+    command: python3 validate.py
```

Comment:
Now we have duplication of: 

test/mnist/package/client/train.py etc
and 
test/mnist/client/train.py etc. 

There are updates in the latter in develop that needs to be ported to the new package folder structure. Then it should be deleted?

Reply:
Updated.
- Remove duplicate code
- Updated package method with better verbosity.